ntbs-service/Migrations/20191015205633_PopulatePostcodeLookup.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using CsvHelper;
+using Microsoft.EntityFrameworkCore.Migrations;
+using ntbs_service.Helpers;
+using ntbs_service.Models;
+
+namespace ntbs_service.Migrations
+{
+    public partial class PopulatePostcodeLookup : Migration

```

Comment:
Thanks, this makes a lot of sense! Can you drop a comment in, explaining why we don't reference this seed data in the ntbsContext?